apps/frontend/src/components/global/groups/GroupModal.vue

```diff
@@ -209,13 +209,13 @@ export default class GroupModal extends Vue {
       this.groupInfo = newGroup();
     }
     this.dialog = false;
+    await GroupsModule.FetchGroupData();
     SnackbarModule.notify(`Group Successfully Saved`);
```

Comment:
isn't it possible for some of these commands esp the axios commands to potentially fail?  we should probably be wrapping a bunch of this in a try/catch instead of always sending out that 'group saved' notification.

Reply:
If the group creation throws an error, that will happen first and go to the snackbar I believe. I'll just add an extra try/catch in case that isn't what actually happens.